cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -393,12 +400,37 @@ private boolean sampleLoadingFinished(Map<TopicPartition, Long> endOffsets) {
       for (Map.Entry<TopicPartition, Long> entry : endOffsets.entrySet()) {
         long position = _consumer.position(entry.getKey());
         if (position < entry.getValue()) {
-          LOG.trace("Partition {} is still lagging. Current position: {}, LEO: {}", entry.getKey(),
+          LOG.debug("Partition {} is still lagging. Current position: {}, LEO: {}", entry.getKey(),
                     position, entry.getValue());
           return false;
         }
       }
       return true;
     }
+
+    private void prepareConsumerOffset() {
```

Comment:
Would it be possible to add comments to this method to explain why we need preparation here? -- i.e. what exactly do we do in this method / what is the benefit of it.